clio-util/src/main/scala/org/broadinstitute/clio/util/auth/AuthUtil.scala

```diff
@@ -72,6 +97,7 @@ object AuthUtil extends ModelAutoDerivation with LazyLogging {
   def getCredsFromServiceAccount(
     serviceAccount: ServiceAccount
   ): Either[Throwable, OAuth2Credentials] = {
+    logger.info("Getting credentials from provided service-account-json")
```

Comment:
Why 'provided'?

Reply:
you can pass a service-account-json in on the command line.

Reply:
Then maybe it should say "Getting credentials from the service-account-json on the command line" or something?

Reply:
Same question about debug logging.